devDocs/startupShutdown.md

```diff
@@ -30,9 +30,23 @@
 1. An external command which kills the process (terminates unsafely) 
 1. Windows shutting down (terminates unsafely) (uses `wx.EVT_END_SESSION`)
 
-## Exit hooks/triggers
+## Technical notes
 
-### In process: `triggerNVDAExit`
+These notes are aimed at developers, wishing to understand technical aspects of the NVDA start and exit.
+
+1. We want only one NVDA instance running at a time, so that it can't interact with itself.

```

Comment:
Would you be able to write on this? I'm not really aware of these mechanisms/problems.

Reply:
You can pretty much just add those three points:
```suggestion
1. No more than one NVDA process instance should be running time. Interactions with itself could cause severe issues, some (non-exhaustive list) examples of sub-systems where this would be a problem:
   - NVDA config files
   - Global (OS level) keyboard hook
   - Changed / incompatible in-process code

```